warehouse/templates/email/removed-as-collaborator.body.txt

```diff
@@ -0,0 +1,16 @@
+Your role in the {{ project }} project has been removed by {{ submitter }}.
+
+{% if role == 'Owner' %}
+You are no longer a project owner. This means that you can no longer add other
+collaborators, upload releases for the project, delete files and releases, or
+delete the entire project.
+{% endif %}
+
+{% if role == 'Maintainer' %}
+You are no longer a project maintainer. This means that you can no longer add
+other collaborators, upload releases for the project, delete files and releases,
+or delete the entire project.
+{% endif %}
+
+If this was a mistake, you can reply to this email directly to communicate with
+the PyPI administrators.
```

Comment:
These emails are now coming from `noreply@pypi.org`, so this message in all these templates will need to be revised, see https://github.com/pypa/warehouse/pull/3844 for an example.